setup/src/main/java/io/jenkins/jenkinsfile/runner/JenkinsfileRunnerRule.java

```diff
@@ -46,7 +54,16 @@ protected ServletContext createWebServer() throws Exception {
         server = new Server(queuedThreadPool);
 
         WebAppContext context = new WebAppContext(warDir.getPath(), contextPath);
-        context.setClassLoader(getClass().getClassLoader());
+        String javaVersion = System.getProperty("java.version");
+
+//        if (javaVersion.startsWith("1.")) {
+//            context.setClassLoader(getClass().getClassLoader());
+//        } else {
+            ClassLoader platform = (ClassLoader) ClassLoader.class.getMethod("getPlatformClassLoader").invoke(null);
```

Comment:
Can we have a comment explaining the intend a bit here?

Reply:
Yes please!

Reply:
IIUC, the goal here is get exactly the same classloader as in previous JDK versions. Since this piece of code is executed always and not only with java11, what will happen if this code is executed with java8? The `getPlatformClassLoader` is only available from java9 (if I'm not wrong), so it will throw an exception